Makefile

```diff
@@ -87,6 +90,7 @@ assets_back := $(assets).bak
 
 back-dependencies:
 	$(GODEP) ensure
+	$(MAKE) -C $(DEPENDENCIES_DIRECTORY)/gopkg.in/bblfsh/client-go.v2 dependencies
```

Comment:
Could we put `CGO_ENABLED=1` only here?
That way main `make build` would run with `CGO_ENABLED=0`...

Reply:
but we need CGO_ENABLED for make build, don't we?

Reply:
I locally tried `CGO_ENABLED=1` only for `client-go`, and then `CGO_ENABLED=0` for the rest.
The rest already was working with `CGO_ENABLED=0` so I wonder if it would still work :dagger:

Reply:
not sure how you managed it. but we are building an binary of the app. The app is linked with c code. If you set cgo=0 uast.go would be excluded by build constraints.

Reply:
you're right, you can disregard this suggestion :dancer: